Bump the flate2 dependency to help apt with updates from
buster to bullseye (Closes: 990317)
Index: object/Cargo.toml
===================================================================
--- object.orig/Cargo.toml
+++ object/Cargo.toml
@@ -76,7 +76,7 @@ optional = true
 default-features = false
 
 [dependencies.flate2]
-version = "1"
+version = "1.0.13"
 optional = true
 
 [dependencies.hashbrown]
